test/unit/node_test.rb

```diff
@@ -58,6 +58,16 @@ class NodeTest < ActiveSupport::TestCase
       assert_equal nodes.length, Tag.find_nodes_by_type('awesome', ['note', 'page']).length
     end
   end
+  
+  test 'node mysql native fulltext search returning two tag-based matches' do
+    assert Node.count > 0
+    if ActiveRecord::Base.connection.adapter_name == 'Mysql2'
+      nodes = Node.search(query: 'awesome',query: 'cool', limit: 1000)
```

Comment:
Ah, hmm, i wonder if here one query would override the other. Perhaps its' best to just submit a single query for, say, `awesome cool`?